src/Env/Config/Config.php

```diff
@@ -42,28 +50,25 @@ public function getFiles()
     {
         $origin = $this->getOrigin();
 
-        yield ($origin instanceof \SplFileInfo) ? $origin : new \SplFileInfo($origin);
+        yield ($origin instanceof SplFileInfo) ? $origin : new SplFileInfo($origin);
```

Comment:
Detail, but why those changes ? i'm not fan of aliasing things from the global namespace.